<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AirportWidget</class>
 <widget class="QWidget" name="AirportWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="1">
    <widget class="QLineEdit" name="searchLineEdit"/>
   </item>
   <item row="1" column="2">
    <widget class="QLabel" name="searchInLabel">
     <property name="text">
      <string>in</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="searchLabel">
     <property name="text">
      <string>Search</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QComboBox" name="searchComboBox"/>
   </item>
   <item row="0" column="0" colspan="4">
    <widget class="QTableView" name="tableView"/>
   </item>
   <item row="2" column="0" colspan="4">
    <widget class="QPushButton" name="newAirportPushButton">
     <property name="text">
      <string>Add New Airport</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
